sway-lib-std/src/u256.sw

```diff
@@ -574,12 +572,14 @@ impl core::ops::Multiply for U256 {
                 let (overflow_of_b_to_a_2, b) = b.overflowing_add(result_d_c.upper).into();
                 let (overflow_of_b_to_a_3, b) = b.overflowing_add(overflow_of_c_to_b_2).into();
 
-                Self::from((
-                    self.b * other.c + result_b_d.upper + overflow_of_b_to_a_3 + overflow_of_b_to_a_2 + overflow_of_b_to_a_1 + overflow_of_b_to_a_0,
-                    b,
-                    c,
-                    result_d_d.lower,
-                ))
+                Self::from(
+                    (
```

Comment:
This change seems unnecessary. I think the formatter should leave it alone in this case.